pkg/operator/controllers/workaround/workaround_controller.go

```diff
@@ -54,7 +54,6 @@ func NewReconciler(log *logrus.Entry, kubernetescli kubernetes.Interface, config
 func (r *WorkaroundReconciler) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.Result, error) {
 	instance, err := r.arocli.AroV1alpha1().Clusters().Get(ctx, arov1alpha1.SingletonClusterName, metav1.GetOptions{})
 	if err != nil {
-		r.log.Error(err)
```

Comment:
Agree, plus we should make this consistent across the operator if the cluster object is necessary.  
i.e https://github.com/Azure/ARO-RP/blob/master/pkg/operator/controllers/azurensg/azurensg_controller.go#L53-L55